dotnet/samples/GettingStartedWithTextSearch/Step1_Web_Search.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.Data;
+using Microsoft.SemanticKernel.Plugins.Web.Bing;
+using Microsoft.SemanticKernel.Plugins.Web.Google;
+
+namespace GettingStartedWithTextSearch;
+
+/// <summary>
+/// This example shows how to create and use a <see cref="ITextSearch"/>.
+/// </summary>
+public class Step1_Web_Search(ITestOutputHelper output) : BaseTest(output)
+{
+    /// <summary>
+    /// Show how to create a <see cref="BingTextSearch"/> and use it to perform a search.
+    /// </summary>
+    [Fact]
+    public async Task BingSearchAsync()
+    {
+        // Create an ITextSearch instance using Bing search
+        var textSearch = new BingTextSearch(apiKey: TestConfiguration.Bing.ApiKey);
+
+        var query = "What is the Semantic Kernel?";
+
+        // Search and return results
+        KernelSearchResults<string> searchResults = await textSearch.SearchAsync(query, new() { Top = 4 });
+        await foreach (string result in searchResults.Results)
+        {
+            Console.WriteLine(result);
+        }
+    }
+
+    /// <summary>
+    /// Show how to create a <see cref="GoogleTextSearch"/> and use it to perform a search.
+    /// </summary>
+    [Fact]
+    public async Task GoogleSearchAsync()
+    {
+        // Create an ITextSearch instance using Google search
+        var textSearch = new GoogleTextSearch(
+            searchEngineId: TestConfiguration.Google.SearchEngineId,
+            apiKey: TestConfiguration.Google.ApiKey);
+
+        var query = "What is the Semantic Kernel?";
+
+        // Search and return results
+        KernelSearchResults<string> searchResults = await textSearch.SearchAsync(query, new() { Top = 4 });
+        await foreach (string result in searchResults.Results)
+        {
+            Console.WriteLine(result);
+        }
+    }
+
+    /// <summary>
+    /// Show how to create a <see cref="BingTextSearch"/> and use it to perform a search
+    /// and return results as a collection of <see cref="BingWebPage"/> instances.
+    /// </summary>
+    [Fact]
+    public async Task SearchForWebPagesAsync()
+    {
+        // Create an ITextSearch instance
+        ITextSearch textSearch = this.UseBingSearch ?
+            new BingTextSearch(
+                apiKey: TestConfiguration.Bing.ApiKey) :
+            new GoogleTextSearch(
+                searchEngineId: TestConfiguration.Google.SearchEngineId,
+                apiKey: TestConfiguration.Google.ApiKey);
```

Comment:
The comments and the casting below only refer to bing, but we have a switch here for google too.  Should this be here?